app/modules/topics/constants.js

```diff
@@ -2,6 +2,6 @@
 
 export const ENDPOINT = '/topics';
 export const USERS_ENDPOINT = '/users';
-export const TOPICS_SUB_ENDPOINT = '/relationships/topics';
+export const TOPICS_SUB_ENDPOINT = '/topics';
 export const INCLUDE = 'include';
 export const USER = 'user';
```

Comment:
These don't need to be constant. The endpoint might change but the relationships probably won't.